.vscode/extensions.json

```diff
@@ -4,5 +4,6 @@
 		"openra.oraide-vscode",
 		"openra.vscode-openra-lua",
 		"EditorConfig.EditorConfig",
+		"macabeus.vscode-fluent",
```

Comment:
Thanks for including my `vscode-fluent` in the recommendations. It made my day happier 😊